styles/zen-browser-docs/catppuccin.user.less

```
@@ -0,0 +1,125 @@
+==/UserStyle== */
+
+@-moz-document domain("zen-browser.app") {
+  :root[data-theme="light"],
+  .light {
+    #catppuccin(@lightFlavor);
+  }
+
+  :root[data-theme="dark"],
+  .dark {
+    #catppuccin(@darkFlavor);
+  }
+
+  #catppuccin(@flavor) {
+    @rosewater: @catppuccin[@@flavor][@rosewater];
+    @flamingo: @catppuccin[@@flavor][@flamingo];
+    @pink: @catppuccin[@@flavor][@pink];
+    @mauve: @catppuccin[@@flavor][@mauve];
+    @red: @catppuccin[@@flavor][@red];
+    @maroon: @catppuccin[@@flavor][@maroon];
+    @peach: @catppuccin[@@flavor][@peach];
+    @yellow: @catppuccin[@@flavor][@yellow];
+    @green: @catppuccin[@@flavor][@green];
+    @teal: @catppuccin[@@flavor][@teal];
+    @sky: @catppuccin[@@flavor][@sky];
+    @sapphire: @catppuccin[@@flavor][@sapphire];
+    @blue: @catppuccin[@@flavor][@blue];
+    @lavender: @catppuccin[@@flavor][@lavender];
+    @text: @catppuccin[@@flavor][@text];
+    @subtext1: @catppuccin[@@flavor][@subtext1];
+    @subtext0: @catppuccin[@@flavor][@subtext0];
+    @overlay2: @catppuccin[@@flavor][@overlay2];
+    @overlay1: @catppuccin[@@flavor][@overlay1];
+    @overlay0: @catppuccin[@@flavor][@overlay0];
+    @surface2: @catppuccin[@@flavor][@surface2];
+    @surface1: @catppuccin[@@flavor][@surface1];
+    @surface0: @catppuccin[@@flavor][@surface0];
+    @base: @catppuccin[@@flavor][@base];
+    @mantle: @catppuccin[@@flavor][@mantle];
+    @crust: @catppuccin[@@flavor][@crust];
+    @accent: @catppuccin[@@flavor][@@accentColor];
+
+    color-scheme: if(@flavor = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --zen-paper: @mantle !important;
+    --zen-dark: @subtext1 !important;
+
+    --color-fd-background: @mantle !important;
+    --color-fd-popover: @mantle !important;
+    --color-fd-secondary: @base !important;
+    --color-fd-accent: @surface0 !important;
+    --color-fd-card: @crust !important;
+    --color-fd-border: @surface0 !important;
+    --color-fd-ring: @overlay0 !important;
+
+    --color-fd-primary: @peach !important;
+    --color-fd-muted: fade(@subtext1, 5%) !important;
+
+    --color-fd-foreground: @subtext1 !important;
+    --color-fd-accent-foreground: @subtext1 !important;
+    --color-fd-card-foreground: @subtext1 !important;
+    --color-fd-primary-foreground: @subtext1 !important;
+    --color-fd-secondary-foreground: @subtext1 !important;
+    --color-fd-popover-foreground: @subtext1 !important;
+    --color-fd-muted-foreground: @subtext1 !important;
+
+    --color-blue-500: @blue !important;
+    --color-red-500: @red !important;
+    --color-orange-500: @peach !important;
+
+    .text-coral,
+    .zen-link,
+    .extra a,
+    .text-\[\#fe846b\],
+    .text-\[\#f7a74b\],
+    .text-\[\#f76f53\] {
+      color: @peach !important;
+    }
```

Instead of using `peach` here I think we should use `accent` for Zen's accent color. I think these are all the right places to change, except maybe `--color-orange-500`?
```suggestion
    --color-fd-primary: @accent !important;
    --color-fd-muted: fade(@subtext1, 5%) !important;

    --color-fd-foreground: @subtext1 !important;
    --color-fd-accent-foreground: @subtext1 !important;
    --color-fd-card-foreground: @subtext1 !important;
    --color-fd-primary-foreground: @subtext1 !important;
    --color-fd-secondary-foreground: @subtext1 !important;
    --color-fd-popover-foreground: @subtext1 !important;
    --color-fd-muted-foreground: @subtext1 !important;

    --color-blue-500: @blue !important;
    --color-red-500: @red !important;
    --color-orange-500: @accent !important;

    .text-coral,
    .zen-link,
    .extra a,
    .text-\[\#fe846b\],
    .text-\[\#f7a74b\],
    .text-\[\#f76f53\] {
      color: @accent !important;
    }
```